### PR TITLE
Add an iv value filter for renamer

### PR DESCRIPTION
**Pull request by pxli168**
User can filter the pokemon for rename by iv, default for rename all(iv value is 0).

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -162,9 +163,10 @@ def rename_pokemons(self):
             name = name.replace("%name", pokemon_name)
             name = name[:12]
 
-            if pokemon['nickname'] == "NONE" \
-               or pokemon['nickname'] == pokemon_name \
-               or (pokemon['nickname'] != name and self.config.overwrite):
+            if (pokemon['nickname'] == "NONE" \
+                or pokemon['nickname'] == pokemon_name \
+                or (pokemon['nickname'] != name and self.config.overwrite)) \
+                and (iv_percent > self.config.iv_value):
```
Comment (tegob) [inline main.py]: No need for `()` here.

File: main.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def init_config(self):
         parser.add_argument("--format", default="%ivsum, %atk/%def/%sta")
         parser.add_argument("-l", "--locale", default="en")
 	parser.add_argument("-d", "--delay", type=int, default=4)
+	parser.add_argument("-iv", "--iv_value", type=int, default=0)
```
Comment (tegob) [inline main.py]: IV stands for “Individual Value” so “IV value” makes no sense. Let’s remove `-iv` and make it `--iv` only?

File: main.py (hunk 1 above)
Comment (pxli168) [inline main.py]: Thanks for advice, I was now to python, it seems the priority of or is higher than and.

File: main.py (hunk 2 above)
Comment (pxli168) [inline main.py]: It is fine with me. I will rebase the code then.

**Pull request merged**